plugin/trino-resource-group-managers/src/main/resources/db/migration/mysql/V3__add_selectors.sql

```diff
@@ -1,10 +1,12 @@
 CREATE TABLE IF NOT EXISTS selectors (
+     id BIGINT NOT NULL AUTO_INCREMENT,
```

Comment:
I believe these files should never be mutated. @hashhar can we have a test?
For example use previous version to populate the DB state and use current to update it?

Reply:
Firstly,  these tables will not be changed after trino first startup unless updated  artificially. And if we update table by adding primary key, mysql will automatically update the historical rows with adding the primary key value. So , there is nothing effect on  previous version DB state.

Reply:
The whole https://github.com/trinodb/trino/tree/master/plugin/trino-resource-group-managers/src/main/resources/db/migration/mysql is used with Flyway, it's a reply-log of DB schema changes, and so individual changes shouldn't change.

@hashhar does Flyway record checksum of the changes files, would it detect an accidental modification?

Reply:
What happens then?

Reply:
> > does Flyway record checksum of the changes files, would it detect an accidental modification?
> 
> It does.

@aalbu  we need a test ensuring no accidental modifications are done.
Could be as benign as someone reformatting large number of files, or Flyway changes checksum result in a corner case. https://github.com/trinodb/trino/pull/10586#discussion_r784099903 would catch any such issue, let's plan on doing it.

cc @hashhar

Reply:
Flyway will verify checksum of the  table beginning with V. If I change the V3__add_selectors.sql file and upgrade trino to this changed version,  trino server start failed  with a checksum exception.  
But Flyway will not verify checksum of the  table beginning with R. If i rename V3__add_selectors.sql to R__add_selectors.sql, and upgrade trino to this changed version , it just print a warn log '**_WARN    main    org.flywaydb.core.internal.sqlscript.DefaultSqlScriptExecutor   DB: Table 'selectors' already exists (SQL State: 42S01 - Error Code: 1050)_**' in server.log. And the trino server start successfully.
So, I wonder if we could rename the V Flyway table to R table?